data/ReadMe.md

```diff
@@ -55,7 +55,11 @@ To make an obs_test defects file from the bias frame generated above, from the o
 
     setup -r .
     data/utils/defectsFromBias.py data/input/bias/bias.fits.gz
-    mv defects_c0.fits data/input/defects/
+    mv defects_c0.ecsv $OBS_TEST_DATA_DIR/test/defects/0/19700101T000000.ecsv
+
+The defects are persisted in the ingested for because ingestion depends on pipe_tasks which introduces a circular dependency.
```

Comment:
I don't understand "are persisted in the ingested for because..."; please rewrite this text.

Also the text breaks up the example commands in a way I don't understand. The command below seems orphaned.

Reply:
Hopefully the new text looks better.